### PR TITLE
IDE-956 IDE "Move File" only works for one file at a time

### PR DESCRIPTION
**Pull request by dehilsterlexis**
Signed-off-by: David de Hilster <david.dehilster@lexisnexis.com>

### CODE REVIEW

**Comment by dehilsterlexis** [comment]
@GordonSmith for review

File: wlib/RepositoryView.h [hunk 1]
```diff
@@ -371,8 +371,17 @@ class CRepositoryViewT
...
+            TreeNode::RefreshChildren(target, m_Root);
+            std::map<std::wstring, bool> modLabels;
+            for (IAttributeVector::const_iterator itr = attrs.begin(); itr != attrs.end(); ++itr)
+            {
+                std::_tstring from = itr->get()->GetModuleQualifiedLabel();
+                if (modLabels.find(from) == modLabels.end())
+                    TreeNode::RefreshChildren(from, m_Root);
+                modLabels[from] = true;
+            }
+        }
```
Comment (GordonSmith) [inline wlib/RepositoryView.h:384]: You will need to refresh on failure now as well (as some may have moved successfully?)

File: comms/DiskRepository.cpp [hunk 2]
```diff
@@ -716,10 +717,13 @@ class CDiskRepository : public CRepositoryBase
...
                 attribute->Refresh(false, attribute);
-                return true;
+                status = true;
+            }
+            else {
+                status = false;
             }
         }
-        return false;
+        return status;
```
Comment (GordonSmith) [inline comms/DiskRepository.cpp:726]: Might be better to return an array of failures, then display a message?

Comment (dehilsterlexis) [reply comms/DiskRepository.cpp:726]: @GordonSmith did think of that as well. Can do...

File: wlib/RepositoryView.h (hunk 1 above)
Comment (dehilsterlexis) [reply wlib/RepositoryView.h:384]: @GordonSmith yes, you are right! Will add...

Comment (GordonSmith) [reply wlib/RepositoryView.h:407]: @dehilsterlexis did this get added?

**Comment by GordonSmith** [comment]
@richardkchapman this is ready to pull.

File: wlib/RepositoryView.h (hunk 1 above)
Comment (dehilsterlexis) [reply wlib/RepositoryView.h:407]: @GordonSmith yes, this was added in my latest push.

**Comment by dehilsterlexis** [comment]
@GordonSmith for review

**Pull request merged**